src/concrete/ml/sklearn/linear_model.py

```diff
@@ -347,6 +355,13 @@ def _get_training_quantized_module(
             fit_bias=self.fit_intercept,
         )
 
+        # Enable the underlying FHE circuit to be composed with itself
+        # This feature is used in order to be able to iterate in the clear n times without having
+        # to encrypt/decrypt the weight/bias values between each loop
+        configuration = Configuration(composable=True)
+
+        composition_mapping = {0: 2, 1: 3}
```

Comment:
don't love this part but I don't see how to properly indicate which input quantizers should be used for which input quantizers

Reply:
so that's an interesting point. I took some time to check if output order is deterministic or not in ONNX, but I couldn't find a definitive answer. It looks like it is (for example, `torch.onnx.export` has a parameter `output_names` that asks for an ordered list of output names), but still, there nothing clear about it in the documentation (ex: https://discuss.pytorch.org/t/are-there-any-guarantees-on-the-output-ordering-of-onnx-model-beyond-topological-sort/151920) 

however, I don't know how we could actually check that + I believe we've always assumed it is, so yes I'll add some checks for the shape but maybe not more than that